modules/EvseV2G/v2g_ctx.cpp

```diff
@@ -492,6 +482,20 @@ bool add_service_to_service_list(struct v2g_context* v2g_ctx, const struct iso2_
     return true;
 }
 
+void check_and_remove_service_in_list(struct v2g_context* v2g_ctx, uint16_t service_id) {
```

Comment:
Looking at the function's counterpart `add_service_to_service_list` and what it does, I would rather go with a function name like `remove_service_from_service_list` or slightly more verbose `remove_service_from_service_list_if_exsits`